index.html

```diff
@@ -446,6 +446,9 @@
 ,{"system":"HeavyAI","date":"2022-07-01","machine":"c6a.4xlarge, 500gb gp2","cluster_size":1,"comment":"Previous names: OmniSci, mapD. Many queries cannot run due to errors and limitations.","tags":["C++","column-oriented"],"load_time":572.633,"data_size":50887437386,"result":[[6.525,0.022,0.029],[0.301,0.042,0.04],[0.287,0.095,0.093],[2.572,0.039,0.04],[null,null,null],[null,null,null],[7.327,0.093,0.097],[0.244,0.043,0.038],[null,null,null],[null,null,null],[null,null,null],[null,null,null],[2.939,0.295,0.294],[null,null,null],[null,null,null],[null,null,null],[null,null,null],[4.716,3.91,3.955],[null,null,null],[0.154,0.083,0.106],[14.426,0.07,0.071],[null,null,null],[null,null,null],[null,null,null],[2.276,0.258,0.272],[null,null,null],[null,null,null],[null,null,null],[null,null,null],[1.832,1.64,1.602],[null,null,null],[null,null,null],[null,null,null],[14.811,0.494,0.497],[null,null,null],[null,null,null],[1.941,0.255,0.255],[5.457,0.172,0.283],[0.476,0.269,0.256],[14.239,0.179,0.178],[3.992,0.112,0.112],[1.031,0.116,0.116],[1.365,0.089,0.088]],"source":"heavyai/results/c6a.4xlarge.json"}
+,{"system":"Hologres","date":"2025-05-21","machine":"16CU","cluster_size":8,"tags":["managed","PostgreSQL compatible","column-oriented"],"load_time":321,"data_size":19923251230,"result":[[0.200416,0.065979,0.018724],[0.091093,0.021522,0.018513],[0.113267,0.021928,0.020564],[0.093743,0.02033,0.025053],[0.13627,0.068549,0.066047],[0.286671,0.131856,0.126948],[0.090226,0.018593,0.018884],[0.096455,0.020777,0.019963],[0.166384,0.144269,0.08159],[0.220452,0.103766,0.0953],[0.143972,0.03618,0.031931],[0.143136,0.036803,0.038698],[0.258057,0.11785,0.109818],[0.330193,0.178884,0.157638],[0.408758,0.122565,0.151418],[0.168366,0.055936,0.057439],[0.429207,0.331269,0.334562],[0.278222,0.243465,0.228009],[0.979158,0.748187,0.788534],[0.077484,0.015684,0.022219],[0.305593,0.026173,0.020405],[0.364327,0.025624,0.024306],[0.579937,0.026864,0.027952],[0.876137,0.048044,0.049257],[0.273008,0.116704,0.04663],[0.184577,0.117629,0.122771],[0.198478,0.049778,0.069377],[0.410839,0.030764,0.028845],[1.926,0.875615,0.802255],[0.318147,0.235507,0.227222],[0.247218,0.066038,0.070967],[0.372834,0.108745,0.111291],[0.74956,0.58827,0.634527],[1.15019,0.86232,0.851895],[1.2923,0.948734,0.950828],[1.64079,1.52869,1.58315],[0.1828,0.039328,0.030409],[0.105002,0.028956,0.035281],[0.085786,0.033011,0.027931],[0.114458,0.096552,0.044727],[0.094894,0.029659,0.028727],[0.101223,0.030141,0.096246],[0.082924,0.025755,0.026889]],"source":"hologres/results/128core.json"}
```

Comment:
Let's revert all changes to index.html. There is a job in CI which does that automatically once the PR is merged.